### PR TITLE
Corrects the application serving from /claimstatus

### LINKED ISSUE #167: Fix static assets in Test env
### Description

We need the site loading properly when accessed via the CA EDD VPN. Currently the images and styling are not being displayed.

### Acceptance Criteria

  - [ ] images and styling load properly when accessed by EDD on Test

### PR DESCRIPTION
**Pull request by lomky**
Properly serves the applications from `/claimstatus` using the `basePath` and `assetPrefix` configuration settings.

 - assetPrefix is only set in production settings (allows Storybook to run properly in dev mode)
 - Sets the storybook build assets to match the assetPrefix
 - Moves the app itself back to `/` to allow the basePath to handle it

Resolves #167 

Related to #122 &  #141

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,4 +58,4 @@ To develop or preview our front end components start up the Storybook server:
 yarn storybook
 ```
 
-The storybook live on `main` can be viewed [on chromatic](https://607065233c37c900212f37e7-ulpsyhmkjm.chromatic.com).
```
Comment (lomky) [inline README.md:68]: turns out these links are per-build. Updating to the app link

File: components/ClaimCardBody.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export const ClaimCardBody: React.FC<ClaimCardBodyProps> = ({
           ))}
         </ul>
       </div>
-      <Link href="/claimstatus" locale={router.locale === 'en' ? 'es' : 'en'}>
```
Comment (lomky) [inline components/ClaimCardBody.tsx:51]: Move back to `/` to let basePath handle it

**Review by kevmo — COMMENTED** [review]
LGTM! 🔥🚀🌌

File: components/Header.tsx [hunk 3]
```diff
@@ -29,7 +29,12 @@ export const Header: React.FC = () => {
       </Navbar>
       <Navbar className="justify-content-between" expand="lg" variant="light">
         <Navbar.Brand target="_blank" rel="noopener noreferrer" href="https://edd.ca.gov">
-          <img src="/images/edd-logo-2-Color.svg" height="60" width="171" className="d-inline-block align-top mr-5" />
+          <img
+            src="/claimstatus/images/edd-logo-2-Color.svg"
+            height="60"
+            width="171"
+            className="d-inline-block align-top mr-5"
+          />
```
[the remark this answers is not in the document]
Comment (lomky) [reply components/Header.tsx:37]: thank the linter! 🤖

**Review by kevmo — APPROVED** [review]
All systems go. ⌚ 🚀🌌

**Pull request merged**